tests/fixtures/bigquery_fixtures.py

```diff
@@ -974,6 +1016,11 @@ def seed_bigquery_integration_db(bigquery_integration_engine) -> None:
         (3, 'admin-account@example.com', 'Monthly Report', 2021, 10, 100),
         (4, 'admin-account@example.com', 'Monthly Report', 2021, 11, 100);
         """,
+        """
+        INSERT INTO fidesopstest.customer_profile VALUES
+        (1, STRUCT('customer-1@example.com', '555-123-4567'), '123 Example Street, Exampletown, NY 12345'),
```

Comment:
We assume using STRUCT instead of JSON, so let's re-ticket support for JSON just in case

Reply:
https://ethyca.atlassian.net/browse/LJ-438